src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/handlers/CcdCallbackHandler.java

```diff
@@ -197,18 +195,6 @@ private void setUnsavedFieldsOnCallback(Callback<SscsCaseData> callback) {
         }
     }
 
-    public static String getBenefitByCode(String code) {
```

Comment:
Why this method removed? is it not being used?

Reply:
It's not used, It was removed [here](https://github.com/hmcts/sscs-bulk-scan/pull/871/files#diff-712695420b7c798abad9d3c91fa884f14fe354b02f427374175da94123303414R53).
generateBenefitCode is used which does something similar apart from throw an BenefitMappingException instead of returning an empty string.